### ISSUE TITLE
consents, ethics and security

### ISSUE THREAD

**Issue by raynamharris** [dcppc/2018-july-workshop#8]
There have been a lot of questions about consents, ethics and security from across the teams.   Cathy Laurie and Alastair Thomson (DataSTAGE) propose a joint presentation on this topic with some specific examples from TOPMed so everyone understands why they are important, what they mean and the implications for the Data Commons. This presentation and subsequent Q&A would be recorded for consortium-wide dissemination.

**Comment by sarahcn** [comment]
Yes! I'm a member of the TOPMed Data Coordinating Center with a particular interest in ethical, legal, and social issues (ELSI). Just to note Cathy Laurie and I have started working on materials for the presentation.

**Comment by raynamharris** [comment]
Join from PC, Mac, Linux, iOS or Android: https://zoom.us/j/749719774

Or iPhone one-tap :
    US: +16699006833,,749719774# or +14086380968,,749719774# 
Or Telephone:
    Dial(for higher quality, dial a number based on your current location): 
        US: +1 669 900 6833 or +1 408 638 0968 or +1 646 876 9923 
    Meeting ID: 749 719 774
    International numbers available: https://zoom.us/u/9yTTAcnd
